internal/values/validate.go

```diff
@@ -18,6 +20,21 @@ import (
 // Returns true if valid. Otherwise, false.
 func (c *SlurmCluster) Validate(ctx context.Context) error {
 	logger := log.FromContext(ctx)
+	// PartitionConfiguration
+	{
+		if c.PartitionConfiguration.ConfigType == "custom" {
+			scanner := bufio.NewScanner(strings.NewReader(c.PartitionConfiguration.RawConfig))
+			for scanner.Scan() {
+				line := scanner.Text()
+				if !strings.HasPrefix(line, "PartitionName") {
```

Comment:
Shouldn't we skip empty lines? (or lines consisting of blank characters)

Reply:
why do u want to have empty lines here?

Reply:
If the partition configuration is long, the user may want to split it into blocks separated by empty lines

Reply:
Changed. Now only if string started with PartitionName we will put it in config